src/Compilers/CSharp/Test/Semantic/Semantics/UserDefinedConversionTests.cs

```diff
@@ -1647,5 +1647,266 @@ private static C<object> M2()
                 //         return (C<object>) M1();
                 Diagnostic(ErrorCode.ERR_NoExplicitConv, "(C<object>) M1()").WithArguments("void", "C<object>").WithLocation(9, 16));
         }
+
+        [Fact, WorkItem(56646, "https://github.com/dotnet/roslyn/issues/56646")]
+        public void LiftedConversion_InvalidTypeArgument01()
```

Comment:
>LiftedConversion_InvalidTypeArgument01 [](http://example.com/codeflow?start=20&length=38)

It looks like verification should be disabled on desktop for test using fake special types.